app/assets/stylesheets/components/_filters.scss

```diff
@@ -1,11 +1,59 @@
-.filters__search-term {
-  max-width: 155px;
+.filters {
+  .nav {
```

Comment:
I guess strictly we're getting into a layout file here as this is styling elements outside of the component?